cmd/compute-domain-kubelet-plugin/checkpoint.go

```diff
@@ -12,14 +12,16 @@ type Checkpoint struct {
 }
 
 type CheckpointV1 struct {
-	PreparedClaims PreparedClaims `json:"preparedClaims,omitempty"`
+	PreparedClaims    PreparedClaims    `json:"preparedClaims,omitempty"`
+	PreparedRawClaims PreparedRawClaims `json:"preparedRawClaims,omitempty"`
```

Comment:
I opted for introducing a new top-level key for storing the ResourceClaim API objects. The type/name `PreparedClaims` was already taken.

Let's find a better name than `PreparedRawClaims`. `PreparedResourceClaims`? `PreparedClaimObjects`?

Reply:
I now repurposed the old name `PreparedClaims`.

What was previously called `PreparedClaims` is now called `PreparedDevicesByClaimUID`.